### PR TITLE
RAIN-5582: If applied it .snyk file which will ignore org.codehaus.pl…

### PR DESCRIPTION
**Pull request by PranotiUplight**
If applied it will add .snyk file which will ignore org.codehaus.plexus:plexus-utils vulnerability.

### CODE REVIEW

**Review by ff-sdesai — APPROVED** [review]

File: .snyk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.19.0  # do not change version, see here for details: https://tendrilinc.slack.com/archives/C02JFTB4550/p1700083772441639?thread_ts=1699577140.305169&cid=C02JFTB4550
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JAVA-ORGCODEHAUSPLEXUS-31522:
+    - '*':
+        reason: 'Unreachable paths.'
+        expires: 2024-07-05T23:59:59Z
+        created: 2024-06-05T11:00:002Z
```
Comment (ff-sdesai) [inline .snyk:9]: We should suppress for at least 3 months,otherwise we'll have to update it again in next month

**Review by ff-sdesai — CHANGES_REQUESTED** [review]
Check comment

File: .snyk (hunk 1 above)
Comment (PranotiUplight) [reply .snyk:9]: Over slack security team (Andriy Zelenyuk ) has suggested that we can set expiry date 1 month from now. they have opened ticket with Snyk team to check this issue. If we get any solution we can fix this issue.

Comment (ff-sdesai) [reply .snyk:9]: Got it

**Review by ff-sdesai — APPROVED** [review]

**Pull request merged**